### PR TITLE
refactor: common repository c# code style

### LINKED ISSUE #84: Repository code style
Hi i'm trying to implement couple enhancement into this repository but faced with couple code style problems.
It reaaaaly hard to jump into code when it bad formated. Right now there is total mess in couple areas:

1. Where to place `{`. Some code written in java style, some code in c#. Sometime both this aproach live in really same file.
2. Single file agreement. It really not really usefull when you have more than one class in single file. 

Also there is couple light problems with library design:

1. To much static stuff, which could be easily replaced by DI. It not hard to fix actually. 
2. Total count of unit tests - 0. At this point nothing to say, but threre is already issue for these (#9).

In a fact there is a reference to aspnet contributing guide which is not followed.
I understand that this repository is mostly playground for experiments, but such problems make it really hard to perform any reasonable contribution in it for anyone except author.

I'm not blaming anyone for this. I just try to help made this tools production ready (I have big plan for this technology). 

So i propose to perform some refactor for this code. 

1. Reformat all code according to common aspnet standarts (4 space, private field with _, c# style of `{` bla-bla). 
2. Extract all classes and interfaces into separate files
3. Remove dependency `Prerenderer` static class inside of tag helper to some `IPrerenderer` interface, which be injected via standart DI process (and could be mocked for testing purpose). And remove all possible static accessors and classes  for better testability and extensibility.
4. Add unit tests for all this stuff (well this issue already exist).

Any thoughts on this @SteveSandersonMS ?

### PR DESCRIPTION
**Pull request by laskoviymishka**
Closes #84
Refactored:

1. Common code formatting
2. Extract files where is possible

Also apply default xproj transformation. Not sure regards to this, but any visual studio with preview tooling done this against this solution, so i think its really necessary (and in a fact without it solution cannot build when you have different from `JavaScriptServices.sln` folder name :sweat_smile: )

### CODE REVIEW

**Comment by SteveSandersonMS** [comment]
Thanks so much for this! It's merged. Everything looks a lot more consistent now - great job!

I made a few further tweaks (cb289fd3) which I hope you don't mind. I'm happy to provide explanations if needed.

**Comment by laskoviymishka** [comment]
All this just a matter of taste :)
I really like this expression body feature, but it just syntax sugar, so it's okay 👌. I'll continue to work under new features now. I have a lot of ideas how improve all this stuff. And now it much easier to do :+1:

**Comment by laskoviymishka** [comment]
Wait for more and more issues and PRs

File: src/Microsoft.AspNetCore.AngularServices/PrimeCacheHelper.cs [hunk 1]
```diff
@@ -3,42 +3,56 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Html;
```
Comment (laskoviymishka) [inline src/Microsoft.AspNetCore.AngularServices/PrimeCacheHelper.cs]: Sorry to @NTaylorMullen, same as PR #75

**Pull request merged**